lib/rules/time-min-milliseconds/README.md

```diff
@@ -0,0 +1,53 @@
+# time-min-milliseconds
+
+Limit the minimum number of milliseconds for time values.
+
+```css
+a { animation: slip-n-slide 150ms linear; }
+/**                         ↑
+ *                  This time */
+```
+
+This rule checks `transition-duration`, `transition-delay`, `animation-duration`, `animation-delay`, and those times as they manifest in the `transition` and `animation` shorthands.
+
+## Options
+
+`int`: Minimum number milliseconds for time values.
+
+For example, with `100`:
+
+The following patterns are considered warnings:
+
+```css
+a { animation: 80ms; }
+```
+
+```css
+a { transition-duration: 0.08s; }
+```
+
+```css
+a { transition: background-color 6ms linear; }
+```
+
+```css
+a { animation: horse-dance 1s linear 0.01s; }
```

Comment:
I find this to be an ambiguous example. A reader might reasonably come away thinking both `1s` and `0.01s` are not ok. Let's just have one time value in each example, since no text specifies which particular value is the violator.

Same applies to the example below.